setup/docker/build.sh

```diff
@@ -13,12 +13,13 @@ tmp="./setup/docker/tmp"
 
 mkdir -p ${tmp}
 
-cat ./setup/unix/traccar.xml | awk '/web.path/ && !modif { printf("    <entry key=\"web.debug\">true</entry>\n"); modif=1 } {print}' > ${tmp}/traccar.xml
+cat ./setup/traccar.xml | awk '/web.path/ && !modif { printf("    <entry key=\"web.debug\">true</entry>\n"); modif=1 } {print}' > ${tmp}/traccar.xml
```

Comment:
Makes sense to me. @oudenaar, can you please fix the comment in the pull request.